pootle/apps/pootle_project/models.py

```diff
@@ -184,7 +184,9 @@ class Project(models.Model, CachedTreeItem, ProjectURLMixin):
         ('auto', _('Automatic detection (slower)')),
         ('gnu', _('GNU style: files named by language code')),
         ('nongnu', _('Non-GNU: Each language in its own directory')),
-    )
+        ('none', _('Do not scan or create directories '
+                   '(Select this option if you wish to manage files '
+                   'using pootle_fs)')))
```

Comment:
Please keep the closing paren on its own line, and add keep the trailing comma too.